src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -293,6 +296,26 @@ public void handle(Message<String> event) {
         registerQueueCheck(modConfig);
     }
 
+    private void registerHttpRequestHandler(RedisquesConfiguration modConfig){
+        log.info("Enable http request handler: " + modConfig.getHttpRequestHandlerEnabled());
```

Comment:
Couldn't we move the body of this method to the RedisquesHttpRequestHandler. Because the class RedisQues is already too large. As I see vertx and modConfig are already passed to the RedisquesHttpRequestHandler.